bob/mkosi.extra/usr/bin/init-firewall.sh

```diff
@@ -188,6 +203,10 @@ iptables -A $CHAIN_MAINTENANCE_IN -j RETURN
 ###########################################################################
 # (9) MAINTENANCE_OUT: Outbound rules for Maintenance Mode
 ###########################################################################
+# Block Flashbots protect tx endpoints during maintenance
+iptables -A $CHAIN_MAINTENANCE_OUT -p tcp -d $FLASHBOTS_TX_IP1 -j DROP
+iptables -A $CHAIN_MAINTENANCE_OUT -p tcp -d $FLASHBOTS_TX_IP2 -j DROP
```

Comment:
We can drop additional tcp filter and combine ips into one argument 

```suggestion
iptables -A $CHAIN_MAINTENANCE_OUT -d $FLASHBOTS_TX_IP1,$FLASHBOTS_TX_IP2 -j DROP
```

Reply:
`iptables -A $CHAIN_MAINTENANCE_OUT -d $FLASHBOTS_TX_IP1 -d $FLASHBOTS_TX_IP2 -j DROP` would also work, whatever looks better for you